assets/packages/adminbasics/src/parts/globalMethods.js

```diff
@@ -112,7 +112,26 @@ const globalWindowMethods = {
             if(e.target != this) return;
             $('#accordion').find('.panel-collapse.collapse').not('#'+$(this).attr('id')).collapse('hide');
         });
-    }
+    },
+    /**
+     * Validates that an end date is not lower than a start date
+     */
+    validateEndDateHigherThanStart: (startDatePicker, endDatePicker, errorCallback) => {
```

Comment:
duplicate ;)

Reply:
This is not a build, it's in src/ folder. So still duplicate. I think.

Reply:
Sorry, I'm wrong.